tests/setup.go

```diff
@@ -89,9 +90,12 @@ func fireCommandAndGetRESPParser(conn net.Conn, cmd string) *clientio.RESPParser
 
 //nolint:unused
 func runTestServer(ctx context.Context, wg *sync.WaitGroup) {
-	config.IOBufferLength = 16
-	config.Port = 8739
-	config.WriteAOFOnCleanup = true
+
+	config.ResetConfig()
+	config.DiceConfig.Network.IOBufferLength = 16
+	config.DiceConfig.Server.Port = 8739
+	config.DiceConfig.Server.PersistenceEnabled = true
```

Comment:
How are these flags different from current functionality implementation?
`PersistenceEnabled` and `WriteAOFOnCleanup`?

Reply:
`PersistenceEnabled`: database will periodically or continuously save its state to disk, ensuring that data can be recovered in case of a crash or restart.

`WriteAOFOnCleanup`: this is meant to control whether the write AOF while performing clean up operations.